lib/std/special/docs/main.js

```diff
@@ -550,6 +550,8 @@
                     console.trace("TODO non-null optional value printing");
                     return "TODO";
                 }
+            case typeKinds.Bool:
+                return token(value, tokenKinds.Null, wantHtml);
```

Comment:
Should there be a new `tokenKinds` for boolean?

Reply:
It's used for `true` and `false` in langref. It should probably be renamed.